app/policies/review_policy.rb

```diff
@@ -0,0 +1,21 @@
+class ReviewPolicy
+  def initialize(book:, reviewer:)
+    @book, @reviewer = book, reviewer
+  end
+
+  def authorized_for_review?
+    if book_reviewers.include? reviewer
```

Comment:
I've made some changes to this class as per your other comments but basically I'm trying to stay away from adding behaviour to AR models. If the upcoming changes aren't any help, I'm happy to implement that suggestion